erdpy/tests/test_cli_contracts.sh

```diff
@@ -120,6 +120,31 @@ testCleanContracts() {
     assertFileDoesNotExist ${SANDBOX}/myfunding-rs/output/myfunding-rs.abi.json || return 1
 }
 
+testVerifyContract(){
+    echo "testVerifyContract"
+
+    nohup python3 local_verify_server.py >/dev/null 2>&1 &
+    sleep 1
+
+    query_response=$(curl -s localhost:7777/verify -X POST)
+
+    command_response=$(${ERDPY} contract verify erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th \
```

Comment:
Perhaps we can use an example address that is a smart contract. E.g. one from here:

https://devnet-explorer.elrond.com/accounts/erd1testnlersh4z0wsv8kjx39me4rmnvjkwu8dsaea7ukdvvc9z396qykv7z7/contracts